src/index.ts

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) WarnerMedia Direct, LLC. All rights reserved. Licensed under the MIT license.
 // See the LICENSE file for license information.
 
-export * from './WireMock';
-export * from './WireMockAPI';
+export * from './externalTypes';
```

Comment:
Since these types are exported at the index level and module level, if any consumer was importing them at module level, the imports will break. This may require us to release as a "breaking change".

Reply:
yeah, i was debating that

the version bump will be a follow-up change so we can make that a major version update